cloudformation/compute-fleet-hit-substack.cfn.yaml

```diff
@@ -365,7 +365,7 @@ Resources:
               if [ -f /opt/parallelcluster/.bootstrapped ]; then
                 installed_version=$(cat /opt/parallelcluster/.bootstrapped)
                 if [ "${!parallelcluster_version}" != "${!installed_version}" ]; then
```

Comment:
Currently `.bootstrapped` file use pcluster version instead of cookbook version. You are right, maybe I should include cookbook version inside `.bootstrapped` file?